server/doky-front/src/components/formComponents/SingleFileUploader.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useId, useState} from "react";
+import {useAddToast} from "../Toasts";
+import {uploadDocument} from "../../api/documents";
+
+const SingleFileUploader = ({
+                              documentId
+                            }) => {
+
+  const [file, setFile] = useState(null)
+  const id = useId()
+  const addToast = useAddToast();
+
+  const handleFileChange = (e) => {
+    if (e.target.files && e.target.files[0]) {
+      setFile(e.target.files[0]);
+    }
+  }
+
+  const handleUpload = async () => {
+    console.log("Document id " + documentId)
```

Comment:
* Clean up console.log's
* Remove or move to separate hook/function error handling logic